packages/form/src/controls/controls.ts

```diff
@@ -172,6 +172,18 @@ export class InputControl extends AbstractControl implements InputControlProps {
       this.value = properties.value ? properties.value : null;
     }
   }
+  
```

Comment:
valueChangeObservable() {
  const observable = new Observable((subscriber) => {
    const internalCallbackOnValueChange = (value) => {
      subscriber.next(value);
    }
    this.changeListener.add(internalCallbackOnValueChange);

    return function unsubscribe() {
      this.changeListener.remove(internalCallbackOnValueChange);
    };
  });

  return observable;
}